packages/smithy/smithy/lib/src/serialization/xml/smithy_xml_plugin.dart

```diff
@@ -232,7 +232,7 @@ class SmithyXmlPlugin implements SerializerPlugin {
       }
       return els;
     } else if (node is XmlText) {
-      return node.text;
+      return node.value;
```

Comment:
I assume this change is fine for all versions of xml we support?

Reply:
Oof - good catch. It's not. And since 6.3.0 requires Dart 2.19, we're not testing anything lower than that. Bumped constraint to `6.3.0` exclusively.